Doc/library/ctypes.rst

```diff
@@ -266,6 +266,17 @@ Fundamental data types
 (1)
    The constructor accepts any object with a truth value.
 
+Additionally, if IEC 60559 compatible complex arithmetic (Annex G) is supported, following
+complex types are available:
+
++----------------------------------+---------------------------------+-----------------+
+| ctypes type                      | C type                          | Python type     |
++==================================+=================================+=================+
+| :class:`c_double_complex`        | :c:expr:`double complex`        | complex         |
```

Comment:
Maybe put that in the table above like this:

```rst
| :class:`c_double_complex` (2)    | :c:expr:`double complex`        | complex         |
...
(2)
    If IEC 60559 compatible complex arithmetic (Annex G) is supported.
```

Reply:
I would like to avoid reformatting the whole table.

Reply:
In the future, we might also have more rows, e.g., `c_float_complex` and `c_longdouble_complex` (maybe `c_float128_complex` but only if Python supports `__float128` is supported at some point (currently it requires `-mfloat128` but it might become the norm in modern compilers, but it also depends on the CPU arch)).

In addition complex types are not necessarily supported by default, which makes them a bit different from the other types, thus it makes sense to have a separate table.

Reply:
I think you can resolve this. If this is accepted, it's justified to add the other types too.